synfig-studio/src/gui/widgets/widget_vector.cpp

```diff
@@ -315,6 +316,29 @@ Widget_Vector::show_all_vfunc()
 	show();
 }
 
+bool Widget_Vector::on_key_press_event(GdkEventKey* key_event)
+{
+  if(key_event->keyval == GDK_KEY_Tab)
+  { 
+    if(entry_x->is_focus()) { 
+		entry_y->grab_focus() ;
+	} 
+	else { 
+		Widget_Vector::activate() ;
+	}
+  }
+  if((key_event->keyval == GDK_KEY_ISO_Left_Tab)) {
```

Comment:
As it was not so obvious for us, what about add an inline comment after `{`: `// Shift+Tab`?